recipes/qwt/all/conanfile.py

```diff
@@ -65,7 +60,9 @@ def requirements(self):
     def build_requirements(self):
```

Comment:
could you move build_requirements after validate() please?

Also:
- move config_options & configure before requirements
- use rm_safe to delete fPIC in configure and bump required_conan_version to 1.53.0
- use export_conandata_patches in export_sources
- extract apply_conandata_patches and call it at the beginning of build()